2014/11/23 12:19:39 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "/var/www/blog/assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in /var/www/framework/web/CAssetManager.php:138
Stack trace:
#0 /var/www/framework/web/CAssetManager.php(123): CAssetManager->setBasePath('/var/www/blog/a...')
#1 /var/www/framework/web/CAssetManager.php(217): CAssetManager->getBasePath()
#2 /var/www/framework/gii/GiiModule.php(142): CAssetManager->publish('/var/www/framew...')
#3 /var/www/framework/base/CComponent.php(111): GiiModule->getAssetsUrl()
#4 /var/www/framework/base/CModule.php(105): CComponent->__get('assetsUrl')
#5 /var/www/framework/gii/views/layouts/main.php(5): CModule->__get('assetsUrl')
#6 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#7 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#8 /var/www/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/framew...', Array, true)
#9 /var/www/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 /var/www/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 /var/www/framework/web/CBaseController.php(206): COutputProcessor->run()
#12 /var/www/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 /var/www/framework/gii/views/layouts/column1.php(7): CBaseController->endContent()
#14 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#15 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#16 /var/www/framework/web/CController.php(784): CBaseController->renderFile('/var/www/framew...', Array, true)
#17 /var/www/framework/gii/controllers/DefaultController.php(47): CController->render('login', Array)
#18 /var/www/framework/web/actions/CInlineAction.php(49): DefaultController->actionLogin()
#19 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#20 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#21 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 /var/www/framework/web/CWebApplication.php(282): CController->run('login')
#23 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#25 /var/www/blog/index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/blog/index.php?r=gii/default/login
---
2014/11/23 12:20:08 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "/var/www/blog/assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in /var/www/framework/web/CAssetManager.php:138
Stack trace:
#0 /var/www/framework/web/CAssetManager.php(123): CAssetManager->setBasePath('/var/www/blog/a...')
#1 /var/www/framework/web/CAssetManager.php(217): CAssetManager->getBasePath()
#2 /var/www/framework/gii/GiiModule.php(142): CAssetManager->publish('/var/www/framew...')
#3 /var/www/framework/base/CComponent.php(111): GiiModule->getAssetsUrl()
#4 /var/www/framework/base/CModule.php(105): CComponent->__get('assetsUrl')
#5 /var/www/framework/gii/views/layouts/main.php(5): CModule->__get('assetsUrl')
#6 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#7 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#8 /var/www/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/framew...', Array, true)
#9 /var/www/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 /var/www/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 /var/www/framework/web/CBaseController.php(206): COutputProcessor->run()
#12 /var/www/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 /var/www/framework/gii/views/layouts/column1.php(7): CBaseController->endContent()
#14 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#15 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#16 /var/www/framework/web/CController.php(784): CBaseController->renderFile('/var/www/framew...', Array, true)
#17 /var/www/framework/gii/controllers/DefaultController.php(47): CController->render('login', Array)
#18 /var/www/framework/web/actions/CInlineAction.php(49): DefaultController->actionLogin()
#19 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#20 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#21 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 /var/www/framework/web/CWebApplication.php(282): CController->run('login')
#23 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#25 /var/www/blog/index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/blog/index.php?r=gii/default/login
---
2014/11/23 12:20:09 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "/var/www/blog/assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in /var/www/framework/web/CAssetManager.php:138
Stack trace:
#0 /var/www/framework/web/CAssetManager.php(123): CAssetManager->setBasePath('/var/www/blog/a...')
#1 /var/www/framework/web/CAssetManager.php(217): CAssetManager->getBasePath()
#2 /var/www/framework/gii/GiiModule.php(142): CAssetManager->publish('/var/www/framew...')
#3 /var/www/framework/base/CComponent.php(111): GiiModule->getAssetsUrl()
#4 /var/www/framework/base/CModule.php(105): CComponent->__get('assetsUrl')
#5 /var/www/framework/gii/views/layouts/main.php(5): CModule->__get('assetsUrl')
#6 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#7 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#8 /var/www/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/framew...', Array, true)
#9 /var/www/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 /var/www/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 /var/www/framework/web/CBaseController.php(206): COutputProcessor->run()
#12 /var/www/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 /var/www/framework/gii/views/layouts/column1.php(7): CBaseController->endContent()
#14 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#15 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#16 /var/www/framework/web/CController.php(784): CBaseController->renderFile('/var/www/framew...', Array, true)
#17 /var/www/framework/gii/controllers/DefaultController.php(47): CController->render('login', Array)
#18 /var/www/framework/web/actions/CInlineAction.php(49): DefaultController->actionLogin()
#19 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#20 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#21 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 /var/www/framework/web/CWebApplication.php(282): CController->run('login')
#23 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#25 /var/www/blog/index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/blog/index.php?r=gii/default/login
---
2014/11/23 12:20:12 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "/var/www/blog/assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in /var/www/framework/web/CAssetManager.php:138
Stack trace:
#0 /var/www/framework/web/CAssetManager.php(123): CAssetManager->setBasePath('/var/www/blog/a...')
#1 /var/www/framework/web/CAssetManager.php(217): CAssetManager->getBasePath()
#2 /var/www/framework/gii/GiiModule.php(142): CAssetManager->publish('/var/www/framew...')
#3 /var/www/framework/base/CComponent.php(111): GiiModule->getAssetsUrl()
#4 /var/www/framework/base/CModule.php(105): CComponent->__get('assetsUrl')
#5 /var/www/framework/gii/views/layouts/main.php(5): CModule->__get('assetsUrl')
#6 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#7 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#8 /var/www/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/framew...', Array, true)
#9 /var/www/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div class="con...')
#10 /var/www/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div class="con...')
#11 /var/www/framework/web/CBaseController.php(206): COutputProcessor->run()
#12 /var/www/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#13 /var/www/framework/gii/views/layouts/column1.php(7): CBaseController->endContent()
#14 /var/www/framework/web/CBaseController.php(126): require('/var/www/framew...')
#15 /var/www/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/framew...', Array, true)
#16 /var/www/framework/web/CController.php(784): CBaseController->renderFile('/var/www/framew...', Array, true)
#17 /var/www/framework/gii/controllers/DefaultController.php(47): CController->render('login', Array)
#18 /var/www/framework/web/actions/CInlineAction.php(49): DefaultController->actionLogin()
#19 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#20 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#21 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 /var/www/framework/web/CWebApplication.php(282): CController->run('login')
#23 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/default/log...')
#24 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#25 /var/www/blog/index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/blog/index.php?r=gii/default/login
---
2014/11/23 12:22:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/23 12:55:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 11:53:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 11:54:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 11:54:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 11:54:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 12:16:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 12:40:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/11/30 12:42:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/12/12 00:08:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/12/12 00:10:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/12/12 00:10:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
---
2014/12/12 00:10:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/12/12 00:12:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /var/www/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/framework/web/CWebApplication.php(282): CController->run('')
#13 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/blog/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/blog/index.php?r=gii/model
HTTP_REFERER=http://localhost/blog/index.php?r=gii/default/index
---
2014/12/20 12:49:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)' in /var/www/framework/db/CDbConnection.php:381
Stack trace:
#0 /var/www/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/framework/base/CApplication.php(438): CModule->getComponent('db')
#4 /var/www/framework/db/ar/CActiveRecord.php(623): CApplication->getDb()
#5 /var/www/framework/db/ar/CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 /var/www/framework/db/ar/CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(User))
#7 /var/www/blog/protected/models/User.php(25): CActiveRecord::model('User')
#8 /var/www/blog/protected/components/UserIdentity.php(25): User::model()
#9 /var/www/blog/protected/models/LoginForm.php(52): UserIdentity->authenticate()
#10 /var/www/framework/validators/CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#11 /var/www/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#12 /var/www/framework/base/CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#13 /var/www/blog/protected/controllers/SiteController.php(94): CModel->validate()
#14 /var/www/framework/web/actions/CInlineAction.php(49): SiteController->actionLogin()
#15 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#16 /var/www/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#17 /var/www/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/framework/web/CWebApplication.php(282): CController->run('login')
#19 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController('site/login')
#20 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#21 /var/www/blog/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/blog/index.php?r=site/login
HTTP_REFERER=http://localhost/blog/index.php?r=site/login
---
2014/12/20 13:17:33 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 13:17:33 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 13:17:33 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 13:17:51 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 13:17:51 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 13:17:51 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:15:58 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:15:58 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:15:58 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:17:54 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:17:54 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:17:54 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:17:57 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:17:57 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:17:57 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:18:00 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:18:00 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:18:00 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:21:23 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:21:23 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:21:23 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:22:58 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:22:58 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:22:58 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:22:58 [error] [php] Use of undefined constant ’ - assumed '’' (/var/www/blog/protected/models/Tag.php:99)
Stack trace:
#0 /var/www/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute()
#1 /var/www/framework/base/CModel.php(159): CInlineValidator->validate()
#2 /var/www/framework/db/ar/CActiveRecord.php(794): Post->validate()
#3 /var/www/blog/protected/controllers/PostController.php(73): Post->save()
#4 /var/www/framework/web/actions/CInlineAction.php(49): PostController->actionCreate()
#5 /var/www/framework/web/CController.php(308): CInlineAction->runWithParams()
#6 /var/www/framework/web/filters/CFilterChain.php(133): PostController->runAction()
#7 /var/www/framework/web/filters/CFilter.php(40): CFilterChain->run()
#8 /var/www/framework/web/CController.php(1145): CAccessControlFilter->filter()
#9 /var/www/framework/web/filters/CInlineFilter.php(58): PostController->filterAccessControl()
#10 /var/www/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#11 /var/www/framework/web/CController.php(291): CFilterChain->run()
#12 /var/www/framework/web/CController.php(265): PostController->runActionWithFilters()
#13 /var/www/framework/web/CWebApplication.php(282): PostController->run()
#14 /var/www/framework/web/CWebApplication.php(141): CWebApplication->runController()
#15 /var/www/framework/base/CApplication.php(169): CWebApplication->processRequest()
#16 /var/www/blog/index.php(13): CWebApplication->run()
REQUEST_URI=/blog/index.php?r=post/create
in /var/www/blog/protected/models/Tag.php (99)
in /var/www/blog/protected/models/Post.php (116)
in /var/www/blog/protected/controllers/PostController.php (73)
2014/12/20 15:23:47 [warning] [application] Failed to set unsafe attribute "create_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:23:47 [warning] [application] Failed to set unsafe attribute "update_time" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
2014/12/20 15:23:47 [warning] [application] Failed to set unsafe attribute "author_id" of "Post".
in /var/www/blog/protected/controllers/PostController.php (72)
in /var/www/blog/index.php (13)
